src/palette.st.css

```diff
@@ -50,13 +52,16 @@
   warningMutedHover:	value(Y30);
   dangerMutedHover:	value(R30);
   premiumMutedHover:	value(P30);
+
+  disabledFields: value(GR20);
+  disabledCheck: value(D50);
 }
 
 /* Input colors */
 :vars {
   inputBackground: value(backgroundSecondary);
   inputBorderColor: value(mainMutedHover);
-  inputHover: value(B50);
+  inputHover: value(mainMutedBackground);
```

Comment:
This is a general naming problem. Input hover what? Text color, border color, background color?

Reply:
I agree but this is the wix-style-react naming convention. I got this name from product.